### ISSUE TITLE
Improve Licence Manager description

### ISSUE THREAD

**Issue by jbonnet** [sonata-nfv/son-gkeeper#354]
We need to further detail the LM module.

Diego (TID) proposed the following:

> [...] I think that a discussion on the gatekeeper as the focal point for license management (among other things), how they will be enforced, and the procedures for the different kinds would be more than enough, just to provide a more comprehensive solution that the usual license server approaches that seem to be favored by some of the NFV participants.
>
> I am wondering that we could use the text of the draft we submitted to NFVRG (see attached) as a base, augmenting it with the considerations on licensing. What is more, we could try to battle at the two fronts: NFVRG (extending the draft to include license management considerations), and ETSI NFV (adapting the draft text into a shorter contribution on applying the gatekeeper) and move them in parallel. What do you think?

**Comment by jbonnet** [comment]
Duplicate of #438
